<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 32);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="side_menu_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QFrame {
	border-radius: 10px;
	background-color: rgb(45, 45, 45);
}

QPushButton {
	text-align: left;
	padding: 5px 1px;
    background: transparent;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
	font-weight: bold;
	icon-size: 24px;
}

QPushButton:hover:!pressed {
	border-left: 3px solid green;
	background-color: gray;
}

QPushButton:checked {
	border-left: 3px solid green;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_library">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Библиотека</string>
            </property>
            <property name="icon">
             <iconset resource="../../desu_res.qrc">
              <normaloff>:/icons/data/icons/library.png</normaloff>
              <normalon>:/icons/data/icons/library_filled.png</normalon>:/icons/data/icons/library.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_main">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Главная</string>
            </property>
            <property name="icon">
             <iconset resource="../../desu_res.qrc">
              <normaloff>:/icons/data/icons/main.png</normaloff>
              <normalon>:/icons/data/icons/main_filled.png</normalon>:/icons/data/icons/main.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_shikimori">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Шикимори</string>
            </property>
            <property name="icon">
             <iconset resource="../../desu_res.qrc">
              <normaloff>:/icons/data/icons/shikimori.png</normaloff>:/icons/data/icons/shikimori.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_history">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>История</string>
            </property>
            <property name="icon">
             <iconset resource="../../desu_res.qrc">
              <normaloff>:/icons/data/icons/history.png</normaloff>:/icons/data/icons/history.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>17</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="top_item_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="palette">
       <palette>
        <active/>
        <inactive/>
        <disabled/>
       </palette>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="top_item">
         <property name="styleSheet">
          <string notr="true">QComboBox {
	padding: 5px 1px;
	border-radius: 10px;
	font-weight: bold;
	background-color: rgb(0, 133, 52);
	color: rgb(255, 255, 255);
}


QPushButton {
	padding: 5px 1px;
	border-radius: 10px;
	font-weight: bold;
	background-color: rgb(0, 133, 52);
	color: rgb(255, 255, 255);
}

QPushButton:hover:!pressed {
	border-left: 3px solid white;
}

QPushButton:checked {
	border-left: 3px solid white;
}
QScrollBar {
    background: #292b2e;
    border-radius: 10px;
}
QScrollBar:horizontal {
    height: 14px;
}
QScrollBar:vertical {
    width: 14px;
	margin: 0px 0px 0px 0px;
}
QScrollBar::handle {
    background: rgba(84.000, 86.000, 86.000, 0.737);
    border-radius: 5px;
	min-height: 0px;
}
QScrollBar::handle:hover {
    background: rgba(114.000, 115.000, 115.000, 0.827);
}
QScrollBar::handle:pressed {
    background: rgba(143.000, 145.000, 145.000, 0.933);
}
QScrollBar::sub-page, QScrollBar::add-page {
    background: transparent;
}
QScrollBar::sub-line, QScrollBar::add-line {
    background: transparent;
	subcontrol-position: top;
    subcontrol-origin: margin;
    width: 0px;
    height: 0px;
}</string>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../desu_res.qrc"/>
 </resources>
 <connections/>
</ui>
